### PR TITLE
details about city district

### PR DESCRIPTION
**Pull request by pepadelic**
**Describe the contribution to the taxonomy**

- Incorrect answer identified regarding topic related to geography and other details related to Brno city and it's district Bystrc (considered as good fix for first contribution to taxonomy project and to get familiar with the workflow)
 - knowledge qna.yaml and attribution.txt created as per guidelines, generated synthetic data, trained the model and verified that new model provides correct information

**Input given at the prompt**

```
in what city is bystrc district? 
```

**Response from the original model**

```
| The Bystřice District is located in the region of Central Bohemia, Czech Republic. This district is	│
│ part of the larger Bohemian Plateau, which stretches from Prague to the German border. Bystřice    	│
│ District is known for its picturesque landscapes, including hills, forests, and rivers, as well as 	│
│ historic towns and villages with medieval architecture. Some notable places in the district include	│
│ Bystřice pod Hostýnem, a town famous        

```

**Response from the fine-tuned model**

```
| Bystrc District is located in the city of Brno, Czech Republic. Brno is the second largest city in 	│
│ Czechia and has a population of about 380,000 people. It's known for its rich cultural life and as one │
│ of the most important scientific centers in the country.                                           	│
│                                                                                                    	│
│ The Bystrc district itself was established back in 1925 and covers an area of approximately 6.4 km2	│
│ (or 2.47 sq mi). The district is primarily residential, with some industrial areas as well. It's   	│
│ located near the northwestern part of Brno and can be accessed easily through public transportation or │
│ by car.                                                  	
```

**Contribution checklist**

- [ x] The contribution was tested with `ilab generate`
- [ x] No errors or warnings were produced by `ilab generate`
- [ ] All [commits are signed off](https://github.com/instructlab/taxonomy/blob/main/CONTRIBUTING.md#legal) (DCO)
- [x ] The `qna.yaml` file contains at least 5 `seed_examples`
- [x ] The `qna.yaml` file was [linted](https://yamllint.com) and [prettified](https://onlineyamltools.com/prettify-yaml) ([yaml-validator](https://jsonformatter.org/yaml-validator) can do both)
- [ x] An `attribution.txt` file in the same folder as the `qna.yaml` file
- [ x] Content does not include PII or otherwise sensitive or confidential information
- [x ] Content does not include anything documented in the project's [Avoid these Topics](https://github.com/instructlab/taxonomy/blob/main/docs/SKILLS_GUIDE.md#avoid-these-topics) guidelines

### CODE REVIEW

**Comment by pepadelic** [comment]
not necessarily clear whether the path to knowledge files within taxonomy tree is selected by contributor or maintainer (my understanding is that maintainers take care of creating right path as per DDC classifications at the moment)

**Comment by jjasghar** [comment]
@instructlab-bot precheck

File: knowledge/geography/attribution.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Title of work: Brno-Bystrc
+Link to work: https://cs.wikipedia.org/wiki/Brno-Bystrc
```
Comment (jjasghar) [inline knowledge/geography/attribution.txt:2]: I'm sorry but we can not accept non-english references.

changing this to `en` changes the page, and doesn't have the data you have below.

**Review by jjasghar — COMMENTED** [review]
The source information needs to be English, translations are not accepted.

**Comment by jjasghar** [comment]
Closing per Triage discussion.

**Pull request closed without merging**